src/ai/backend/manager/models/endpoint.py

```diff
@@ -379,6 +384,9 @@ class Meta:
 
     endpoint_id = graphene.UUID()
     image = graphene.String()
+    image_id = graphene.UUID(description="Added at 23.09.9")
+    image_row = graphene.Field(Image, description="Added at 23.09.9")
+    architecture = graphene.String()
```

Comment:
Please add `architecture` description with the version

Reply:
Bulls eye! I just applied the description. thx!